README.md

```diff
@@ -47,49 +47,42 @@ https://github.com/user-attachments/assets/160c721d-a941-440a-bc42-afd419c1df18
 
 ```fish
 λ phase
-Securely manage and sync environment variables with Phase.
-
-⠀⠀⠀⠀⠀⠀⠀⠀⠀⢠⠔⠋⣳⣖⠚⣲⢖⠙⠳⡄⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
-⠀⠀⠀⠀⠀⠀⠀⠀⡴⠉⢀⡼⠃⢘⣞⠁⠙⡆⠀⠘⡆⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
-⠀⠀⠀⠀⠀⠀⢀⡜⠁⢠⠞⠀⢠⠞⠸⡆⠀⠹⡄⠀⠹⡄⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
-⠀⠀⠀⠀⠀⢀⠞⠀⢠⠏⠀⣠⠏⠀⠀⢳⠀⠀⢳⠀⠀⢧⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
-⠀⠀⠀⠀⢠⠎⠀⣠⠏⠀⣰⠃⠀⠀⠀⠈⣇⠀⠘⡇⠀⠘⡆⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
-⠀⠀⠀⢠⠏⠀⣰⠇⠀⣰⠃⠀⠀⠀⠀⠀⢺⡀⠀⢹⠀⠀⢽⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
-⠀⠀⢠⠏⠀⣰⠃⠀⣰⠃⠀⠀⠀⠀⠀⠀⠀⣇⠀⠈⣇⠀⠘⡇⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
-⠀⢠⠏⠀⢰⠃⠀⣰⠃⠀⠀⠀⠀⠀⠀⠀⠀⢸⡀⠀⢹⡀⠀⢹⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
-⢠⠏⠀⢰⠃⠀⣰⠃⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣇⠀⠈⣇⠀⠈⡇⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
-⠛⠒⠚⠛⠒⠓⠚⠒⠒⠓⠒⠓⠚⠒⠓⠚⠒⠓⢻⡒⠒⢻⡒⠒⢻⡒⠒⠒⠒⠒⠒⠒⠒⠒⠒⣲⠒⠒⣲⠒⠒⡲⠀⠀⠀⠀
-⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢧⠀⠀⢧⠀⠈⣇⠀⠀⠀⠀⠀⠀⠀⠀⢠⠇⠀⣰⠃⠀⣰⠃⠀⠀⠀⠀
-⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠘⡆⠀⠘⡆⠀⠸⡄⠀⠀⠀⠀⠀⠀⣠⠇⠀⣰⠃⠀⣴⠃⠀⠀⠀⠀⠀
-⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠹⡄⠀⠹⡄⠀⠹⡄⠀⠀⠀⠀⡴⠃⢀⡼⠁⢀⡼⠁⠀⠀⠀⠀⠀⠀
-⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠙⣆⠀⠙⣆⠀⠹⣄⠀⣠⠎⠁⣠⠞⠀⡤⠏⠀⠀⠀⠀⠀⠀⠀⠀
-⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⠳⢤⣈⣳⣤⣼⣹⢥⣰⣋⡥⡴⠊⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀
-
-Options:
+Securely manage application secrets and environment variables with Phase.
+
+                     @@@             
+              @@@@@@@@@@     
+          @@@@@@@@@@@@@@@@
+       P@@@@@&@@@?&@@&@@@@@P
+     P@@@@#        @&@    @P@@@
+    &@@@#         *@&      #@@@&
+   &@@@5          &@?       5@@@&
+  Y@@@#          ^@@         #@@@J
+  #@@@7          B@5         7@@@#
+  #@@@?         .@@.         ?@@@#
+  @@@@&         5@G          &@@@7
+   #@@@B        @@^         #@@@B
+    B@@@@      .@#        7@@@@B
+     @@@@@@    &.@       P@@@@@7
+       @@@@@@@@@@@@@@@@@@@@@
+          @@@@@@@@@@@@@@@
+             @@@@@@@@
+             @@@   
+
+options:
   -h, --help   show this help message and exit
   --version, -v
                show program's version number and exit
 
 Commands:
-
-    auth             💻 Authenticate with Phase
-    init             🔗 Link your project with your Phase app
-    run              🚀 Run and inject secrets to your app
-    secrets          🗝️ Manage your secrets
-    secrets list     📇 List all the secrets
-    secrets get      🔍 Get a specific secret by key
-    secrets create   💳 Create a new secret
-    secrets update   📝 Update an existing secret
-    secrets delete   🗑️ Delete a secret
-    secrets import   📩 Import secrets from a .env file
-    secrets export   🥡 Export secrets in a dotenv format
-    users            👥 Manage users and accounts
-    users whoami     🙋 See details of the current user
-    users logout     🏃 Logout from phase-cli
-    users keyring    🔐 Display information about the Phase keyring
-    docs             📖 Open the Phase CLI Docs in your browser
-    console          🖥️ Open the Phase Console in your browser
-    update           🆙 Update the Phase CLI to the latest version
+  {auth,init,run,secrets,users,docs,console,update}
```

Comment:
i think we should show all sub-sub eg. `phase secrets export` commands here.

when i have some time i'll figure out how to print the sub-sub commands in argparse.